Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/BocTreeViewControlObjectTest.cs

```diff
@@ -638,7 +642,7 @@ public void TestContextMenuControlObject_OpenDropDownMenuWithDelayGreaterThanTim
 
       var contextMenu = node.GetContextMenu();
 
-      Assert.That (() => contextMenu.Open(), Throws.TypeOf<MissingHtmlException>().With.Message.EqualTo ("Unable to open the menu."));
+      Assert.That (() => contextMenu.Open(), Throws.TypeOf<WebTestException>().With.Message.EqualTo ("Unable to open the menu."));

```

Comment:
The test specifically mentions that this happens when the server needs too long to return the context menu. This means that the error happens when the Projekt messed up and took too long to refresh the page --> Bug --> WebTestException

Reply:
@FlorianDecker `scope["attributeName"]` does not throw a `MissingHtmlException` if the attribute does not exist (simply returns `null`). This would mean that our `WebTestException ("Unable to open the menu.")` thrown if the attribute is null should be alright. We were just thinking that the fact that missing attributes do not cause exceptions might hide some valueable informations.

Reply:
Added [RM-7228](https://re-motion.atlassian.net/browse/RM-7228).